### PR TITLE
Linh

### PR DESCRIPTION
**Pull request by LinhTangTD**
- Fix bugs in current graphs: legend showing incorrect labels
- Add visualizations for different recovery rate (100, 75, 50, 25, 0)
- Add `nb.html` file for better preview

### CODE REVIEW

**Comment by LinhTangTD** [comment]
- Add cost to models
- Fix visualization with total sick + total cost updated in graphs
- Add comment to code

File: Complex SIR model.Rmd [hunk 1]
```diff
@@ -36,10 +36,14 @@ library(ggplot2)
...
-* Number of days since first case $(day = 40)$
+* Treatment A cost $(A_{cost} = 25.00)$
+
+* Treatment B cost $(B_{cost} = 15.00)$
+
+* Number of days since first case $(day = 38)$
 
 ```{r model.norand}
-complex.model.norand = function(N=800, r=0.01, g=0.1, d1=40, c1=0, PTA=0.5, PTB=0.5, Aeff=0.65, Beff=0.45, day = 40){
+complex.model.norand = function(N=800, r=0.01, g=0.1, d1=40, c1=0, PTA=0.5, PTB=0.5, A.eff=0.65, B.eff=0.45, day = -1, A.cost = 25, B.cost = 15){
```
Comment (minceb) [inline Complex SIR model.Rmd:46]: What was your reasoning for not adding testing in the non-random complex model

File: Complex SIR model WITH variability.Rmd [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+  NDAT = tpd + u
+  NDTA = 0
+  NDTB = 0
+  DCA = 0
+  DCB = 0
+  NAT = 0
+  NTA = 0
+  NTB = 0
+  SDC = 0
+  New = min(N - c - d, round(r * hs * d))
```
Comment (minceb) [inline Complex SIR model WITH variability.Rmd:86]: Do you know why this is the function for people who catch the disease? I am a little confused.

**Review by minceb — COMMENTED** [review]
Moreover, it's not random in the complex model with variability. I recognize that is not the case in the excel file for both models. However, I believe it is valuable to make that variable random.

**Pull request merged**